src/components/PodcastCard/PodcastCard.js

```diff
@@ -0,0 +1,46 @@
+import styled from 'styled-components/macro'
+import PropTypes from 'prop-types'
+
+PodcastCard.propTypes = {
+  image: PropTypes.string.isRequired,
+  title: PropTypes.string.isRequired,
+}
+
+export default function PodcastCard({ image, title }) {
+  return (
+    <Card>
+      <Image
+        src={image}
+        alt="Podcast Main Artwork"
+        width="100%"
+        height="clamp(350px, 75%, 800px)"
```

Comment:
This should be just integer values without units (or even css). Please look it up on https://www.smashingmagazine.com/2021/04/humble-img-element-core-web-vitals/

Reply:
Done